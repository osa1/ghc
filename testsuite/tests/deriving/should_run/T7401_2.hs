{-# LANGUAGE EmptyDataDecls #-}

data Foo deriving (Show, Read, Eq, Ord)

main = print $ (undefined :: Foo) `compare` (undefined :: Foo)
