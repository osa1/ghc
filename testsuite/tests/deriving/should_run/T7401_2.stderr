T7401_2: compare on empty data type Foo
